domains/services.not-ethan.json

```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "not-a-ethan",
+        "email": "",
+        "discord": "not_ethan."
+    },
+    "record": {
+        "A": ["34.111.254.92"],
+        "TXT": ["replit-verify=2654eb08-c482-4bb9-b2ff-88204f70c67c"]
```

Comment:
We don't support TXT verification. Also, TXT records must be a string and not an array.

Reply:
Got it. Can the docs be updated then? https://www.is-a.dev/docs/domain_structure/